.claude/commands/check-best-practices.md

```diff
@@ -0,0 +1,118 @@
+# /check-best-practices
+
+Analyze code files for adherence to OpenShift Console conventions and style guide compliance.
+
+## Instructions
+
+You are an expert Senior Principal Software Engineer conducting code reviews for the OpenShift Console codebase. Your role is to ensure code follows established best practices, style guidelines, and maintains consistency across the large React/TypeScript codebase that serves as the frontend for OpenShift, a popular Kubernetes distribution.
+
+### Goal
+Analyze files for compliance with OpenShift Console best practices, style guide adherence, and code quality standards.
+
+## Reference Documentation
+
+**IMPORTANT**: Always read and reference these authoritative sources before analyzing code:
+
+1. **[OpenShift Console Style Guide](../../STYLEGUIDE.md)** - Official coding standards for directory naming, Go, TypeScript/JavaScript, React, and SCSS/CSS
+2. **[AI Context Documentation](../../.ai/context.md)** - Project structure, frontend/backend patterns, and development guidelines
+
+These documents are the single source of truth for OpenShift Console coding standards.
+
+## Analysis Process
```

Comment:
I see this section as overlapping with what would most likely be provided in category specific files, that are treated as the single, authoritative place where a piece of data, configuration, or state is defined, stored, and maintained.

```
console/
├── ARCHITECTURE.md
├── CONVENTIONS.md
├── STYLEGUIDE.md
├── TESTING.md
└── INTERNATIONALIZATION.md
```